packages/sitecore-jss-nextjs/src/services/mutisite-graphql-sitemap-service.ts

```diff
@@ -0,0 +1,91 @@
+import { getSiteRewrite } from '@sitecore-jss/sitecore-jss/site';
+import {
+  GraphQLSitemapService,
+  GraphQLSitemapServiceConfig,
+  languageError,
+  RouteListQueryResult,
+  StaticPath,
+} from './graphql-sitemap-service';
+
+export const sitesError = 'The list of sites cannot be empty';
+
+/**
+ * Configuration options for @see GraphQLSitemapService instances
+ */
+export interface MultisiteGraphQLSitemapServiceConfig extends GraphQLSitemapServiceConfig {
+  /**
+   * Names of the configured sites
+   */
+  sites: string[];
+}
+
+/**
+ * Service that fetches the list of site pages using Sitecore's GraphQL API.
+ * Used to handle multiple sites
+ * This list is used for SSG and Export functionality.
+ * @mixes SearchQueryService<PageListQueryResult>
+ */
+export class MultisiteGraphQLSitemapService extends GraphQLSitemapService {
+  /**
+   * Creates an instance of graphQL sitemap service with the provided options
+   * @param {GraphQLSitemapServiceConfig} options instance
```

Comment:
This type should be updated (and possibly the description on line above)
```suggestion
   * @param {MultisiteGraphQLSitemapServiceConfig} options instance
```